### PR TITLE
small issues found trying to compile examples

### PR DESCRIPTION
**Pull request by tkruse**

### CODE REVIEW

File: CppCoreGuidelines.md [hunk 1]
```diff
@@ -12159,7 +12159,7 @@ When unambiguous, the `T` can be left out of `T{e}`.
 
     complex<double> f(complex<double>);
 
-    auto z = f({2*pi, 1});
+    auto z = f({2 * pi, 1});
```
Comment (cubbimew) [inline CppCoreGuidelines.md]: there's an open issue questioning these spaces around arithmetic operations, #948

Comment (tkruse) [inline CppCoreGuidelines.md]: thanks for the pointer. Not sure if that discussion would have an impact on this location as well. I can remove from this PR and create a separate one if that helps.

**Comment by tkruse** [comment]
added a few more tiny issues, squashed.

File: CppCoreGuidelines.md (hunk 1 above)
Comment (AndrewPardoe) [inline CppCoreGuidelines.md]: @tkruse, could you please remove the 2 * pi, and then we can merge this.

**Comment by tkruse** [comment]
I removed the additional 2 whitespaces around the arithmetic expression as requested.

File: CppCoreGuidelines.md [hunk 2]
```diff
@@ -12670,7 +12672,7 @@ For example
         operator int() { return val; }
     };
 
-    int f(Positive arg) {return arg };
+    int f(Positive arg) {return arg; }
```
Comment (galik) [inline CppCoreGuidelines.md]: Should there be a space before the `return arg;` as the previous example?

File: CppCoreGuidelines.md [hunk 3]
```diff
@@ -12670,7 +12672,7 @@ For example
         operator int() { return val; }
     };
 
-    int f(Positive arg) {return arg };
+    int f(Positive arg) { return arg; }
```
Comment (tkruse) [inline CppCoreGuidelines.md]: added blank before return for consistency

**Comment by AndrewPardoe** [comment]
Thank you! Merging this, but will also manually fix references to <Foo> in the Enforcement section of C.151.

**Pull request merged**